### DISCUSSION TITLE
Is there a method for selecting a date range with the Calendar component?

### DISCUSSION THREAD

**Discussion by harachanx** [shadcn-ui/ui#1508] [Q&A] [answered]
I'm assuming the following behaviors
![DateRangePicker](https://github.com/shadcn-ui/ui/assets/28142346/2a4e5e95-f812-439f-9407-8e5fb5424e14)
https://mui.com/x/react-date-pickers/date-range-picker/#available-components

Waiting for your reply!

**Comment by itaim18** [comment, marked as answer]
Hey, the calender is implemented with [react-day-picker/](https://react-day-picker.js.org/).
You should look on their [implementation](https://react-day-picker.js.org/guides/input-fields#example-range-selection) of range in order to have it in your app.

**Reply by harachanx** [reply]
@itaim18 This is exactly what I wanted.
Thank you.
